packet/templates/extend/email.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+
+{% block head %}
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
```

Comment:
Is there a reason we shouldn’t use CSH material?

Reply:
I don't think so. I think that was just me tired at midnight not thinking. I'll fix that (also need to place it in the <head> tag to begin with